src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -566,7 +566,7 @@ ReactDOMComponent.Mixin = {
           div.innerHTML = `<${type}></${type}>`;
           el = div.removeChild(div.firstChild);
         } else if (props.is) {
-          el = ownerDocument.createElement(type, props.is);
+          el = ownerDocument.createElement(type, {is: props.is});
```

Comment:
See [custom elements spec](https://html.spec.whatwg.org/multipage/scripting.html#custom-elements-customized-builtin-example) for why I changed this. It explains that you create customized built-in elements like so:

```js
/* Method 1: the "is" attribute in the markup
 * React doesn't do this because it calls createElement instead of setting innerHTML
 */
document.body.innerHTML = '<button is="my-button"></button>';

/* Method 2: the "is" option when calling createElement
 * React _needs_ to do this in order for the browser to correctly upgrade an
 * element to be a customized built-in element.
 */
document.createElement('button', {is: 'my-button'});
```